Compiler/Util/Flags.mo

```diff
@@ -761,6 +764,7 @@ constant ConfigFlag POST_OPT_MODULES = CONFIG_FLAG(16, "postOptModules",
     "solveLinearSystem",
     "addScaledVars",
     "removeSimpleEquations",
+    "simplifyComplexFunction",
```

Comment:
A (brief) description of the newly introduced optimization module "simplifyComplexFunction" is missing.